src/python/py/models/README.md

```diff
@@ -217,6 +217,20 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options use_qdq=1
 ```
 
+#### LoRA Models
```

Comment:
Can you add a [reference](https://github.com/microsoft/onnxruntime-genai/pull/955#discussion_r1807486816) to this new section in the table of contents?